npm/qsharp/ux/circuit-vis/formatters/inputFormatter.ts

```diff
@@ -71,38 +71,15 @@ const formatInputs = (
 const _qubitInput = (y: number, subscript?: string): SVGElement => {
   const el: SVGElement = text("", leftPadding, y, 16);
 
-  // Create the main text node
-  const mainText = document.createElementNS(
-    "http://www.w3.org/2000/svg",
-    "tspan",
-  );
-  mainText.textContent = "|𝜓";
+  const subtext = subscript
+    ? `<tspan baseline-shift="sub" font-size="65%">${subscript}</tspan>`
```

Comment:
Was this done to avoid using the `document.createElementNS` calls? The function calls seem more "official" than hardcoding the string, but is there a reason to prefer the hardcoded string?

Reply:
It's simpler/shorter/easier to read in this case. Either works.